tests/e2e/cluster_downgrade_test.go

```diff
@@ -207,6 +207,77 @@ func testDowngradeUpgrade(t *testing.T, numberOfMembersToDowngrade int, clusterS
 	assert.Equal(t, beforeMembers.Members, afterMembers.Members)
 }
 
+func TestAutomaticDowngradeCancellationAfterCompletingDowngradingInClusterOf3(t *testing.T) {
```

Comment:
The name is too long.
```suggestion
func TestDowngradeAutoCancelAfterCompletion(t *testing.T) {
```